src/web/mjs/HakuNeko.mjs

```diff
@@ -32,6 +33,7 @@ export default class HakuNeko {
         this._connectors = new Connectors(ipc);
         this._storage = new Storage();
         this._bookmarkManager = new BookmarkManager(this._settings, new BookmarkImporter());
+        this._comicInfoGenerator = new ComicInfoGenerator();
```

Comment:
This instance seems not to be of global significance to be placed in the engine scope, just inject it into the `Storage` constructor, thats the only place where it is really needed.